Development.md

```diff
@@ -57,6 +57,16 @@ To automatically fix issues run `npm run fix` command.
 ## Running Unit Tests
 Run `npm test` to execute the unit tests via [Jest](https://jestjs.io/).
 
+#### Unit testing guideline
+- Component tests should be located in the same directory as the tested component and named `component.spec.ts`.
+- Focus on testing components that are shared across the project (see `src/app/shared`).
```

Comment:
Perhaps it is also worth mentioning that we keep test utils and mocks in the `src/test`.